apps/src/lib/node/ledger/mod.rs

```diff
@@ -406,7 +408,17 @@ fn start_abci_broadcaster_shell(
 
     // Channels for validators to send protocol txs to be broadcast to the
     // broadcaster service
-    let (broadcaster_sender, broadcaster_receiver) = mpsc::unbounded_channel();
+    let (broadcast_sender, broadcast_receiver) = mpsc::unbounded_channel();
+
+    // Channels for validators to send events to an event log, which will be
+    // queried from the node's RPC `/events` endpoint.
+    //
+    // TODO: make the sender a param of `start_abci_broadcaster_shell`;
+    // the sender will be returned by another task which starts the
+    // event log; we will also need yet another task for the rpc
+    // server itself. if we are not a validator, `event_log_sender`
+    // should be `None`
```

Comment:
Shouldn't regular full nodes be providing an `/events` endpoint as well? In which case `event_log_sender` shouldn't be an `Option`